src/javascript/javascript-parser.ts

```diff
@@ -27,6 +27,16 @@ declare class SyntaxError {
   column: number;
 }
 
+export interface Comment {
+  type: string;
+  value: string;
+  loc: SourceLocation;
+}
+
+export interface Program extends EstreeProgram {
+  comments: Comment[];
```

Comment:
Also, I don't see where this is being populated anywhere? Were comments already being attached to Programs and this is just to satisfy the type checker?